ni_measurementlink_service/discovery/__init__.py

```diff
@@ -0,0 +1,6 @@
+"""Client for accessing the MeasurementLink discovery service."""
```

Comment:
Seems like this should be similar to
"""Redirect internal discovery client API to the public API."""

Reply:
This **is** the public API, so I updated the docstring to """Public API for accessing the MeasurementLink discovery service."""